### PR TITLE
 Add nfc in privileged permissions. (bug 1149431)

### PR DESCRIPTION
**Pull request by allstarschh**

### CODE REVIEW

**Comment by allstarschh** [comment]
Also I filed Bug 1149431 for this, not sure if PR is okay or should I send r? to someone on Bugzilla ?

File: appvalidator/constants.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
         'device-storage:videos', 'device-storage:music',
         'device-storage:sdcard', 'feature-detection',
         'input', 'mobileid', 'mobilenetwork', 'speaker-control', 'systemXHR',
-        'tcp-socket'
+        'tcp-socket', 'nfc'
```
Comment (diox) [inline appvalidator/constants.py]: Please insert it after `'mobilenetwork'` to keep this sorted (easier to spot stuff added twice this way)

**Comment by diox** [comment]
Doing a PR is the right workflow for Marketplace contributions. Please amend your commit message and PR description to include the bug number, like this "message (bug xxxx)". Thanks.

**Comment by allstarschh** [comment]
Thanks for the fast reply.
I've moved the position of 'nfc' and updated the commit message with bug id.

Thanks

**Pull request merged**